src/arrays/programs_involving_multidimensional_data/hard/q3/EightByEight.java

```diff
@@ -0,0 +1,43 @@
+package arrays.programs_involving_multidimensional_data.hard.q3;
+
+/*
+Write a Java program that initializes an 8 by 8 2D array with random values from 0 to 10 (inclusive of both values) and
+prints each row in a line. Then write a function called to find that takes a 2d array
+and an index, and prints out the value at that index. Using that, find the value of the element at index 7.
+Consider some examples with an 8 by 8 array. Index 2 points to the element in the first row and the third column. Index
+8 points to the element in the second row and the first column. Index 16 points to the element in the third row and the
+first column.
```

Comment:
We also need to mention that their method should be general enough to work with any input length if we are going to test for that.